packages/@aws-cdk/aws-rds/lib/instance.ts

```diff
@@ -593,6 +593,12 @@ export interface DatabaseInstanceNewProps {
    * @default - None
    */
   readonly s3ExportBuckets?: s3.IBucket[];
+  /**
+   * Indicates whether the DB instance is an internet-facing instance
```

Comment:
However, a period here would be nice 🙂.

```suggestion
   * Indicates whether the DB instance is an internet-facing instance.
```